src/core_modules/capture-core/metaDataStoreLoaders/baseLoader/loadWithContext.js

```diff
@@ -0,0 +1,15 @@
+// @flow
```

Comment:
Why dont we have this being exported from the `/baseLoader/metaDataLoader.js` and we create another file? 

Instead of exporting then `loadMetaData` from the `/baseLoader/metaDataLoader.js` we would export something like `loadMetaDataWithContext`. Is there any advantage by adding this file?

Reply:
I will rename the files (loadWithContext -> loadMetaData, loadMetaData -> loadMetaDataInternal). The advantage is separation of concerns, keeping the context importing and building separate.

Reply:
That makes sense